extension/chrome/elements/pgp_block_modules/pgp-block-error-module.ts

```diff
@@ -32,7 +32,7 @@ export class PgpBlockViewErrorModule {
   }
 
   public handlePrivateKeyMismatch = async (message: Uint8Array) => { // todo - make it work for multiple stored keys
-    const msgDiagnosis = await BrowserMsg.send.bg.await.pgpMsgDiagnosePubkeys({ privateKis: await Store.keysGet(this.view.acctEmail), message });
+    const msgDiagnosis = await PgpMsg.diagnosePubkeys({ privateKis: await Store.keysGet(this.view.acctEmail), message });
```

Comment:
This should not work - PgpMsg does not have OpenPGP.js. Instead of sending PgpMsg to bg script, you should be sending PgpMsg to parent script (typically the main content script), where OpenPGP.js should be imported if not present already.

Reply:
But since the tests worked.. could you verify if this method is being hit in tests? By just adding `throw new Error('Whoops')` ? If not, then we need to add a new test before we proceed with this further. Thanks!

Reply:
I updated the first comment to say that `pgp_block.htm` frames do not have OpenPGP.js loaded.

This is for performance - there may be 50 such frames, and it takes 300ms of compute time to load OpenPGP.js in each frame. It's a fat library.

Reply:
> But since the tests worked.. could you verify if this method is being hit in tests? By just adding `throw new Error('Whoops')` ? If not, then we need to add a new test before we proceed with this further. Thanks!

Now, I know which test is failing and how to hit that method manually, thanks!